src/js/apps/patients/worklist/worklist_app.js

```diff
@@ -72,19 +72,22 @@ export default App.extend({
 
     if (storedState) {
       this.setState(storedState);
+      this.getState().setClinicianId(this.clinicianId);
       this.startFiltersApp();
       return;
     }
 
     this.setState({ id: this.worklistId });
 
+    this.getState().setClinicianId(this.clinicianId);
```

Comment:
This seems simpler to me than going the `initState()` route. But maybe there's a reason we don't want to use a state method here 🤔

Reply:
I'm ok with it.  outside of the `onBeforeStart` it would probably cause real issues of multiple restart/renders